lib/matplotlib/backends/qt_compat.py

```diff
@@ -42,10 +38,6 @@
     QT_API = QT_API_PYQT5
 elif sys.modules.get("PySide2.QtCore"):
     QT_API = QT_API_PYSIDE2
-elif sys.modules.get("PyQt4.QtCore"):
```

Comment:
do we still want these checks and turn them into errors?  Pretty sure that even though we no longer support Qt4, if the user has managed to get them imported Qt5 and Qt6 will be broken.

Reply:
But this section only determines which one we want to attempt. If Qt4 is loaded and conflicts, then when we attempt Qt5/6, it'll fail and move on to another backend?

Reply:
oh, fair point :+1: